### ISSUE TITLE
Make API publicly available 3rd try

### ISSUE THREAD

**Issue by albertkol** [canonical-web-and-design/ubuntu-com-security-api#6]
RT link: https://rt.admin.canonical.com/Ticket/Display.html?id=128604&results=608d25c102559890d8074a12a5cfa24e

**Comment by albertkol** [comment]
We are in the process of choosing another domain name for the API.

Until I have a new domain name the API cannot go public, so I will block this. 

As soon as we have chosen the domain name, I will make all the necessary config changes and issue another ticket to IS.

@SirSamTumless @nottrobin

**Comment by albertkol** [comment]
https://rt.admin.canonical.com/Ticket/Display.html?id=128748&results=caa72f78d099d9be44b6c77ad4c4f9ee

Unblocked! The new ticket is pending with priority: `urgent`.

**Comment by albertkol** [comment]
Deploying the api at https://ubuntu.com/security
